oracle/oracle.go

```diff
@@ -24,252 +27,131 @@ type Oracle interface {
 	IsRunning() bool
 	GetLastSyncTime() time.Time
 	GetPrices() types.Prices
-	GetMarketMap() *marketmaptypes.MarketMap
+	GetMarketMap() mmtypes.MarketMap
```

Comment:
nit: can you move this and all other interfaces to the `interfaces.go` file.

Reply:
done [move interfaces to interface.go](https://github.com/skip-mev/slinky/pull/503/commits/12ba0b0d4f7fca0621b4a5a447631e9f476c61b7)